pkgs/development/compilers/julia/patches/1.9-bin/0001-allow-skipping-internet-required-tests.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/share/julia/test/choosetests.jl b/share/julia/test/choosetests.jl
+index 334ef05..df6d03a 100644
+--- a/share/julia/test/choosetests.jl
++++ b/share/julia/test/choosetests.jl
+@@ -31,6 +31,19 @@ const TESTNAMES = [
+         "smallarrayshrink", "opaque_closure", "filesystem", "download",
+ ]
+ 
++const INTERNET_REQUIRED_LIST = [
++    "Artifacts",
++    "Downloads",
++    "LazyArtifacts",
++    "LibCURL",
++    "LibGit2",
++    "Pkg",
++    "download",
++    "TOML",
++]
++
++const NETWORK_REQUIRED_LIST = vcat(INTERNET_REQUIRED_LIST, ["Sockets"])
++
+ """
```

Comment:
Looks like `filtertests!(tests, "internet_required", INTERNET_REQUIRED_LIST)` has been dropped from the patch?

Reply:
Indeed, nice catch. I just added it back.